src/components/AccessibilityPolicyForm/index.jsx

```diff
@@ -0,0 +1,278 @@
+      status = {
+        alertType: 'danger',
+        alertDialog: (
+          <div>
+            We are currently experiencing high volume. Try again later today or
+             send an email message to <a href={`mailto:${email}`}>{email}</a>
+          </div>
+        ),
+      };
+    } else if (accessibilityStatus.type ===
+        accessibilityActions.ACCESSIBILITY_FORM_SUBMIT_SUCCESS) {
+      status = {
+        alertType: 'success',
+        alertDialog: (
+          <div>
+            Thank you for contacting edX!
+            <br />
+            <br />
+            Thank you for your feedback regarding the accessibility of Studio. We typically respond
+             within one business day (Monday to Friday, 13:00 to 21:00 UTC).
+          </div>
+        ),
+      };
+    }
+
+    return status;
+  }
+
+  validateFormContent(inputType, inputValue) {
+    let feedback = { isValid: true };
+    if (inputType === 'email') {
+      /* eslint-disable max-len */
+      /* eslint-disable no-useless-escape */
+      const emailRegEx = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+      /* eslint-enable max-len */
+      if (!emailRegEx.test(inputValue)) {
+        feedback = {
+          isValid: false,
+          validationMessage: 'Valid email required.',
+        };
+      }
+    } else if (inputType === 'fullName') {
+      if (!inputValue) {
+        feedback = {
+          isValid: false,
+          validationMessage: 'Full name required.',
+        };
+      }
+    } else if (inputType === 'message') {
+      if (!inputValue) {
+        feedback = {
+          isValid: false,
+          validationMessage: 'Message required.',
+        };
+      }
+    }
+
+    return feedback;
+  }
+
+  closeStatusAlert() {
+    this.emailInputRef.focus();
+    this.setState({
+      statusAlertOpen: false,
+    });
+  }
+
+  handleChange(value, name) {
+    if (name === 'email') {
+      this.setState({
+        formEmail: value,
+      });
+    } else if (name === 'fullName') {
+      this.setState({
+        formFullName: value,
+      });
+    } else if (name === 'message') {
+      this.setState({
+        formMessage: value,
+      });
+    }
+  }
+
+  renderStatusAlert() {
+    const status = this.getStatusFields();
+
+    const statusAlert = (
+      <StatusAlert
+        alertType={status.alertType}
+        dialog={status.alertDialog}
+        open={this.state.statusAlertOpen}
+        onClose={this.closeStatusAlert}
+        ref={(input) => { this.statusAlertRef = input; }}
+      />
+    );
+
+    return (
+      <div>
+        {statusAlert}
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <div>
+        <h1 id="a11y-feedback" className={styles['page-header']}>Studio Accessibility Feedback</h1>
+        {this.renderStatusAlert()}
+        <h6>All fileds required.</h6>
```

Comment:
Spelling :)

Also, I don't think `h6` is semantically right for this. Screen readers sometimes navigate by headers and this should probably a div with some styling.